v22.2/cockroach-debug-zip.md

```diff
@@ -11,7 +11,7 @@ The `cockroach debug zip` [command](cockroach-commands.html) connects to your cl
 You can use the [`cockroach debug merge-logs`](cockroach-debug-merge-logs.html) command in conjunction with `cockroach debug zip` to merge the collected logs into one file, making them easier to parse.
 
 {{site.data.alerts.callout_danger}}
-The files produced by `cockroach debug zip` can contain highly [sensitive, identifiable information](configure-logs.html#redact-logs), such as usernames, hashed passwords, and possibly table data. Use the [`--redact-logs`](#redact-sensitive-information-from-the-logs) flag to redact sensitive data from log files and crash reports before sharing them with Cockroach Labs.
+The files produced by `cockroach debug zip` can contain highly [sensitive, personally-identifiable information (PII)](configure-logs.html#redact-logs), such as usernames, hashed passwords, and possibly table data. Use the [`--redact`](#redact-sensitive-information) flag to configure CockroachDB to redact sensitive data when generating the `.zip` file (excluding range keys) if intending to share it with Cockroach Labs.
```

Comment:
> Use the [`--redact`](#redact-sensitive-information) flag to configure CockroachDB to redact sensitive data

I wonder if it's worth discerning between "redact" and "omit" here 🤔. We redact whenever possible, but a good number of `crdb_internal` and `system` table columns we omit entirely. 

Just a thought, feel free to disregard - especially if it might confuse the reader!

Reply:
Just for style points, you might split the callout into two, keeping the first sentence in the danger one and putting the second sentence in a success callout, since it's a solution.

Reply:
Thanks, I can see the utility (and style!) but I think that keeping these two statements together conceptually helps reinforce their connection: I think the two statements would be less effective if delivered separately.